### PR TITLE
Talk about Debian compatibility in the README.

### PR DESCRIPTION
**Pull request by vext01**
Small doc update. OK?

### CODE REVIEW

**Review by ltratt — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ consult the Krun documentation (fetched into `krun/` by `build.sh`), as there
 is a great deal of manual intervention needed to compile a tickless kernel,
 disable Intel P-states, set up `rc.local` etc.
 
+Note that the experiment is designed to run on amd64 machines running Debian 8
+or OpenBSD. Newer versions of Debian do not currently work due to a C++ ABI
+bump which would require us to use a newer C++ compiler.
```
Comment (ltratt) [inline README.md]: Can we say something like "which would require porting the experiment to use a newer C++ compiler" (I'm hoping if we don't say "us" that some kind soul might take it upon themselves to doing the porting!).

Comment (vext01) [reply README.md]: Yeah, but I expect that if someone were to update it, they would not care for OpenBSD.

Updating GCC is easy if you don't care about OpenBSD.

Comment (ltratt) [reply README.md]: OK, so maybe we should say "a newer C++ compiler (i.e. a newer version of GCC or clang)"?

**Comment by vext01** [comment]
Try this.

**Comment by ltratt** [comment]
You don't think it's worth being clear that "newer" could mean GCC or clang?

**Comment by vext01** [comment]
I don't want to be too specific because I don't know the constraints on clang wrt libstdc++.

**Comment by ltratt** [comment]
If we say "e.g." would that be OK? I just think that if I read that, I'd think "only GCC will do", whereas clang is, at the very least, a plausible substitute. [I'm partly thinking that since clang is now better supported on OpenBSD, it might be the lower effort route.]

**Comment by vext01** [comment]
I've revised the wording to include clang as a possible next step.

My hesitancy comes from the fact clang has its own `libc++`, whereas g++ uses `libstdc++` and I'm not sure to what degree you can mix them. It's likely the system C++ libs link libstdc++.

OK to squash?

**Comment by ltratt** [comment]
Works for me! Please squash.

**Comment by vext01** [comment]
splat

**Pull request merged**